buildAndReleaseTask/task.json

```diff
@@ -68,6 +68,14 @@
             "defaultValue": "",
             "helpMarkDown": "The Pulumi Stack against which to run the commands.",
             "required": "true"
+        },
+        {
+            "name": "expectedVersion",
+            "type": "string",
+            "label": "Pulumi version",
+            "defaultValue": "latest",
+            "helpMarkDown": "The Pulumi version that should be used.",
```

Comment:
A couple of examples of the version numbers would be useful in the `helpMarkDown`. Advertising the special keyword `latest` here might be a good idea too.

Reply:
done